src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// Reducer
+const categoriesReducers = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return ['Under Contruction'];
```

Comment:
- [ ] We are expected to return the string 'Under construction', not an array. Kindly fix that.

Reply:
Resolved